cpm

```diff
@@ -1,216 +1,259 @@
-    done
-    echo $i
+	i=0
+	while read -r line; do
+		i=$((i + 1))
+	done
+	echo $i
 }
 
 # replacement for command -v which ignores aliases
 # this is useful for me, don't ask why
 has() {
-    case "$(command -v $1 2>/dev/null)" in
-        alias*|"") return 1
-    esac
+	case "$(command -v "$1" 2>/dev/null)" in
+		alias*|"") return 1
+	esac
 }
 
 _apt() {
-    case "$OP" in
-        install) apt install "$@";;
-        remove) apt remove "$@";;
-        list) apt list --installed;;
-        count) dpkg-query -f '.\n' -W | tot;;
-        update) apt update;;
-        upgrade) apt dist-upgrade;;
-        search) apt search "$1";;
-        show) apt show "$1";;
-        clean) apt autoremove;;
-    esac
+	case "$OP" in
+		install) apt install "$@";;
+		remove) apt remove "$@";;
+		list) apt list --installed;;
+		count) dpkg-query -f '.\n' -W | tot;;
+		update) apt update;;
+		upgrade) apt dist-upgrade;;
+		search) apt search "$1";;
+		show) apt show "$1";;
+		clean) apt autoremove;;
+	esac
 }
 
 _dnf() {
-    case "$OP" in
-        install) dnf install "$@";;
-        remove) dnf remove "$@";;
-        list) dnf list --installed;;
-        count) rpm -qa | tot;;
-        update) dnf check-update;;
-        upgrade) dnf distro-sync;;
-        search) dnf search "$1";;
-        show) dnf info "$1";;
-        clean) dnf autoremove;;
-    esac
+	case "$OP" in
+		install) dnf install "$@";;
+		remove) dnf remove "$@";;
+		list) dnf list --installed;;
+		count) rpm -qa | tot;;
+		update) dnf check-update;;
+		upgrade) dnf distro-sync;;
+		search) dnf search "$1";;
+		show) dnf info "$1";;
+		clean) dnf autoremove;;
+	esac
 }
 
 _nix() {
-    case "$OP" in
-        install) nix-env -iA "$@";;
-        remove) niv-env -e "$@";;
-        list) nix-env -q "$@";;
-        count) nix-env -q | wc -l;;
-        update) nix-channel --update;;
-        upgrade) nix-env -u;;
-        search) nix-env -qa "$@";;
-        show) nix-env -qa --description "$@";;
-        clean) nix-collect-garbage -d;;
-    esac
+	case "$OP" in
+		install) nix-env -iA "$@";;
+		remove) niv-env -e "$@";;
+		list) nix-env -q "$@";;
+		count) nix-env -q | wc -l;;
+		update) nix-channel --update;;
+		upgrade) nix-env -u;;
+		search) nix-env -qa "$@";;
+		show) nix-env -qa --description "$@";;
+		clean) nix-collect-garbage -d;;
+	esac
 }
 
 _pacman() {
-    case "$OP" in
-        install) pacman -S "$@";;
-        remove) pacman -Rs "$@";;
-        list) pacman -Q;;
-        count) pacman -Q | tot;;
-        update) pacman -Sy;;
-        upgrade) pacman -Syu;;
-        search) pacman -Ss $1;;
-        show) pacman -Si $1;;
-        clean) pacman -Rns $(pacman -Qtdq) && pacman -Sc;;
-    esac
+	case "$OP" in
+		install) pacman -S "$@";;
+		remove) pacman -Rs "$@";;
+		list) pacman -Q;;
+		count) pacman -Q | tot;;
+		update) pacman -Sy;;
+		upgrade) pacman -Syu;;
+		search) pacman -Ss "$1";;
+		show) pacman -Si "$1";;
+		clean) pacman -Rns "$(pacman -Qtdq)" && pacman -Sc;;
```

Comment:
I can't test, but I'm very sure that would break with multiple packages (removing the package `meson ninja` instead of the packages `meson` and `ninja`.

Reply:
No it wouldn't.

Reply:
nvm i thought you meant the remove command, not the clean command. yeah it would.